### PR TITLE
tldr page for fuser

### PR DESCRIPTION
**Pull request by davidsiefert**
Didn't see this here, so adding. :)

feedback welcome!

### CODE REVIEW

File: pages/linux/fuser.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# fuser
+
+> display process IDs currently using files or sockets
+
+- identify process using a tcp socket
+
+`fuser -n tcp {{port}}`
+
+*Requires admin privileges, otherwise silently fails (exit code 1, no output)*
```
Comment (igorshubovych) [inline pages/linux/fuser.md]: Don't have this note here. You'd better add short note in description:
```Markdown
> Display process IDs currently using files or sockets.
> Require admin privileges.
```

**Comment by igorshubovych** [comment]
Squash commits.

**Comment by davidsiefert** [comment]
updated and squashed

File: pages/linux/fuser.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# fuser
+
+> Display process IDs currently using files or sockets.
+> Require admin privileges.
+
+- identify process using a tcp socket
```
Comment (igorshubovych) [inline pages/linux/fuser.md]: In command description it is better to write TCP in capitals

**Comment by igorshubovych** [comment]
Overall LGTM. You can fix capitals in description, because it is an abbreviation.

**Comment by igorshubovych** [comment]
Thank you for your contribution!

**Pull request merged**